core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1182,7 +1182,7 @@ write_files:
           -ec \
           'echo decrypting assets
            shopt -s nullglob
-           for encKey in /etc/kubernetes/{ssl,{{ if or (.AssetsConfig.HasAuthTokens) ( and .Experimental.TLSBootstrap.Enabled .AssetsConfig.HasTLSBootstrapToken) }}auth{{end}}}/*.enc; do
+           for encKey in /etc/kubernetes/{ssl,additional-configs,{{ if or (.AssetsConfig.HasAuthTokens) ( and .Experimental.TLSBootstrap.Enabled .AssetsConfig.HasTLSBootstrapToken) }}auth{{end}}}/*.enc; do
```

Comment:
I'd rather prefer `encryption` over `additional-configs` like other directories at the same level, so that we can be more explicit about what are stored under the `encryption` directory - an encryption config.

Reply:
Addressed in a follow-up commit

Reply:
Oops. No, we already have this additional-configs directory for another feature. So Im happy to just rely on it today.